### PR TITLE
Add FAQs from github/conscious-lang-docs

### PR DESCRIPTION
**Pull request by celestehorgan**
Signed-off-by: Celeste Horgan <celeste@cncf.io>
Co-authored-by: Rich Bowen <rbowen@rcbowen.com>

### CODE REVIEW

File: content/faqs.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+
+---
+title: "Frequently asked questions"
+---
+
+
+## What is inclusive naming about?
+
+The software industry sometimes uses certain words, like blacklist, and slave, that may carry a great deal of emotional and historical baggage. We are encouraging projects to choose phrases that clearly communicate the technical meaning, without using metaphors or colloquialisms.
```
Comment (xmulligan) [inline content/faqs.md:9]:
```suggestion
The software industry sometimes uses certain words, like blacklist and slave, that may carry a great deal of emotional and historical baggage. We are encouraging projects to choose phrases that clearly communicate the technical meaning, without using metaphors or colloquialisms.
```

File: content/faqs.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## What terms are we recommending you consider?
+
+We have identified the terms "master/slave" and "whitelist/blacklist" to work on because these are considered most visible and problematic across the industry.
+
+Over time, we may recommend consideration of other terms, such as words that reference mental health, gender, physical handicaps, and several other categories. We may also give tips on avoiding colloquialisms that simply don't translate well or prove a barrier to understanding.
+
+## What are the recommended replacement terms?
+
+See the word replacement list for all terms we recommend replacing.
```
Comment (xmulligan) [inline content/faqs.md:27]: is there a link for this?

File: content/faqs.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Newspeak involved multiple changes but the most essential were:
+
+ * Removing precise words in favor of general words so that it would become difficult to form precise thoughts
+ * Using the opposite word for the true situation to disguise what's really happening
+
+Everything in the conscious language initiative is *the exact opposite* of that.
+
+For example, "Denylist" is both more precise and more accurate than "blacklist";
+
+"Primary" is both more precise and more accurate than "master" in projects that have made that switch.
```
Comment (xmulligan) [inline content/faqs.md:43]:
```suggestion
"Primary" is both more precise and more accurate than "master" in projects that have made the switch.
```

File: content/faqs.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+ * [That Word Black (2014)](http://mcwriting11.blogspot.com/2014/06/that-word-black-by-langston-hughes.html) - blog post by Langston Hughes
+ * [Broken Metaphor: The Master-Slave Analogy in Technical Literature (2007)](https://www.jstor.org/stable/40061475?seq=1) - article in JSTOR, requires free membership to read
+ * [Terminology, Power and Oppressive Language](https://tools.ietf.org/id/draft-knodel-terminology-00.html) - article with recommendations by Mallory Knodel and Niels ten Oever
+ * [Why "master" matters](https://twitter.com/mislav/status/1270388510684598272) - A twitter thread.
+
+## How do I get my organization involved?
+
+* Join our [mailing list](https://groups.google.com/g/inclusivenaming) to receive project updates and invitations to community meetings. 
+* Identify key people in your organization who can represent this effort in an ongoing fashion
+* Read our Implementation Guid and other resources to start making effective changes
```
Comment (xmulligan) [inline content/faqs.md:67]:
```suggestion
* Read our Implementation Guide and other resources to start making effective changes
```

**Review by xmulligan — APPROVED** [review]

File: content/faqs.md (hunk 2 above)
Comment (celestehorgan) [reply content/faqs.md:27]: It's in another PR – there's a bit of cleanup to do after all these merge.

**Pull request merged**